crates/wasi-preview1-component-adapter/src/macros.rs

```diff
@@ -3,23 +3,11 @@
 //! We're avoiding static initializers, so we can't have things like string
 //! literals. Replace the standard assert macros with simpler implementations.
 
-use crate::bindings::wasi::io::streams;
-
 #[allow(dead_code)]
 #[doc(hidden)]
-pub fn print(message: &[u8]) {
-    let _ = unsafe { streams::write(crate::get_stderr_stream(), message) };
-}
-
-/// A minimal `eprint` for debugging.
-#[allow(unused_macros)]
-macro_rules! eprint {
-    ($arg:tt) => {{
-        // We have to expand string literals into byte arrays to prevent them
-        // from getting statically initialized.
-        let message = byte_array_literals::str!($arg);
-        $crate::macros::print(&message);
-    }};
+pub fn println(message: &[u8]) {
+    // FIXME: We need some way to print a message.
+    let _ = message;
```

Comment:
We can run `get_stderr().blocking_write_and_flush()` here now, correct?

Reply:
Yep, this is now done.